src/lib/backend/services/adminService.ts

```
@@ -0,0 +1,67 @@
+      state: response.data.state!,
+      customerAddress: response.data.customerAddress!,
+      orderItems: response.data.orderItems?.map((item) => ({
+        id: item.id!,
+        orderId: item.orderId!,
+        productId: item.productId!,
+        count: item.count!,
+        price: item.price!,
+      })) || [],
+    };
+  }
+} 
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)